### PR TITLE
Short documentation on how to configure CORS

### PR DESCRIPTION
**Pull request by GregDThomas**

### CODE REVIEW

**Comment by dblock** [comment]
Thanks. Just needs to be linked from FAQ.md I think.

File: Docs/faq/CORS.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
+# How does the Waffle Servlet Security Filter work with CORS?
+
+The Servlet Security Filter works fine with CORS. Configuring it depends on your server. 
+
+## Jetty
+If you are using Jetty, then it's a simple case of using the Jetty 
+[`CrossOriginFilter`](http://www.eclipse.org/jetty/documentation/current/cross-origin-filter.html) to configure CORS. 
+The following `web.xml` fragment gives an example;
```
Comment (dblock) [inline Docs/faq/CORS.md]: Maybe a colon or a period instead of a semicolon?

**Comment by GregDThomas** [comment]
OK, I added the link from FAQ.md and changed the ";" for a ":". I also updated the example to explicitly include non-default port for one of the servers so that it's clear that is required.

**Comment by dblock** [comment]
Merged, thank you.

**Pull request merged**